src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/arrival/Arrival.kt

```diff
@@ -45,3 +48,8 @@ enum class LocationType {
   PRISON,
   OTHER
 }
+
+enum class Gender {
+  MALE,
+  FEMALE,
```

Comment:
Might be better to add a comment here that we can only map across male and female, if BASM returns trans, then we pass back null as we need to explicitly capture gender information.